modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]AllowUsersToDefineAPIApplicationEndpoints/EditAPIApplicationEndpoint.testcase

```diff
@@ -35,6 +35,66 @@ definition {
 		}
 	}
 
+	@priority = 3
+	test CanChangeSchemaOfEndpoint {
+		task ("And Given two API Schema in API Application") {
+			SchemaAPI.createAPISchema(
+                apiApplicationERC = "My-app",
+                mainObjectDefinitionERC = "L_API_APPLICATION",
+                name = "firstSchema");
+
+            SchemaAPI.createAPISchema(
+                apiApplicationERC = "My-app",
+                mainObjectDefinitionERC = "L_API_APPLICATION",
+                name = "secondSchema");
+		}
+
+		task ("And Given setting API Schema schema1 in Response Body Schema") {
```

Comment:
Your code look good here, but the rule in headless qa automation test is we need to make the code and the GWT steps consistent,  could you please update `firstSchema` and `secondSchema` to schema1 and schema2? Thanks